transports/azure-storage-queues/multi-storageaccount-support.md

```diff
@@ -32,51 +31,36 @@ A typical implementation uses a single storage account to send and receive messa
+When the number of instances of endpoints are increased, all endpoints continue reading and writing to the same storage account. Once the limit of 2,000 message/sec per queue or 20,000 message/sec per storage account is reached, Azure throttles the message throughput.
 
 ![Single storage account with scaled out endpoints](azure02.png "width=500")
 
-While an endpoint can only read from a single Azure storage account, it can send messages to multiple storage accounts. Configure this by specifying a connection string when message mapping. Each endpoint will have its own storage account to overcome the Azure storage account throughput limitation of 20,000 messages/sec.
-
-Example: Endpoint 1 sends messages to Endpoint 2. Endpoint 1 defines message mapping with a connection string associated with the Endpoint 2 Azure storage account. The same idea applies to Endpoint 1 sending messages to Endpoint 2.
-
-Message mapping for Endpoint 1:
-
-```xml
-<MessageEndpointMappings>
-  <add Messages="Contracts"
-       Namespace="Contracts.Commands.ForEndpoint2"
-       Endpoint="Endpoint2@connection_string_for_endpoint_2" />
-</MessageEndpointMappings>
-```
+While an endpoint can only read from a single Azure storage account, it can send messages to multiple storage accounts. This way one can set up a solution using multiple storage accounts where each endpoint uses its own Azure storage account, thereby increasing message throughput.
 
-Message mapping for Endpoint 2:
+![Scale out with multiple storage accounts](azure03.png "width=500")
 
-```xml
-<MessageEndpointMappings>
-  <add Messages="Contracts"
-       Namespace="Contracts.Commands.ForEndpoint1"
-       Endpoint="Endpoint1@connection_string_for_endpoint_1" />
-</MessageEndpointMappings>
-```
+## Scale Units
 
-Each endpoint uses its own Azure storage account, thereby increasing message throughput.
+Scaleout and splitting endpoints over multiple storage accounts works to a certain extent, but it cannot be applied infinitely while expecting throughput to increase accordingly. Only so much throughput from a single resource or group of resources grouped together is possible.
 
+A suitable techniques to overcome this problem includes resource partitioning and use of scale units. A scale unit is a set of resources with well determined throughput, where adding more resources to this unit does not result in increased throughput. When the scale unit is determined, to improve throughput, create more scale units. Scale units do not share resources.
 
-![Scale out with multiple storage accounts](azure03.png "width=500")
+An example of a partitioned application with a different number of deployed scale units is an application deployed in various regions.
 
+![Scale units](azure04.png "width=500")
 
-partial: aliases
+NOTE: Use real Azure storage accounts because the Azure storage emulator only supports a single fixed account named `devstoreaccount1`.
```

Comment:
> Use real Azure storage accounts because the Azure storage emulator only supports a single fixed account named `devstoreaccount1`.

How about "Use real Azure storage accounts. Do not useAzure storage emulator as it only supports a single fixed account named `devstoreaccount1`."?

Reply:
addressed